### PR TITLE
Move filesystem sensor to standard provider

### PR DESCRIPTION
**Pull request by kunaljubce**
Move filesystem sensor to standard provider, relates to #43641.

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: docs/apache-airflow/howto/operator/file.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@
 FileSensor
 ==========
 
-Use the :class:`~airflow.sensors.filesystem.FileSensor` to detect files appearing in your local
+Use the :class:`~airflow.providers.standard.sensors.filesystem.FileSensor` to detect files appearing in your local
```
Comment (gopidesupavan) [inline docs/apache-airflow/howto/operator/file.rst:25]: Please move this to standard provider docs. ref: https://github.com/apache/airflow/pull/43716

File: docs/apache-airflow/operators-and-hooks-ref.rst [hunk 2]
```diff
@@ -79,7 +79,7 @@ For details see: :doc:`apache-airflow-providers:operators-and-hooks-ref/index`.
    * - :mod:`airflow.sensors.external_task`
      - :doc:`How to use <howto/operator/external_task_sensor>`
 
-   * - :mod:`airflow.sensors.filesystem`
+   * - :mod:`airflow.providers.standard.sensors.filesystem`
```
Comment (gopidesupavan) [inline docs/apache-airflow/operators-and-hooks-ref.rst]: This is not required you can remove this. because it falls under standard provider now.

Comment (kunaljubce) [reply docs/apache-airflow/operators-and-hooks-ref.rst]: @gopidesupavan Removed!

File: docs/apache-airflow/howto/operator/file.rst (hunk 1 above)
Comment (kunaljubce) [reply docs/apache-airflow/howto/operator/file.rst:25]: @gopidesupavan Moved this to `docs/apache-airflow-providers-standard/operators/`.

**Review by romsharon98 — APPROVED** [review]

**Comment by potiuk** [comment]
nice!

**Pull request merged**